unstructured_inference/models/chipper.py

```diff
@@ -46,6 +46,17 @@
         "heatmap_w": 30,
         "source": Source.CHIPPER,
     },
+    "chipperv3": {
+        "pre_trained_model_repo": "unstructuredio/chipper-v3",
+        "swap_head": True,
+        "swap_head_hidden_layer_size": 128,
+        "start_token_prefix": "<s_",
+        "prompt": "<s><s_hierarchical>",
+        "max_length": 1536,
+        "heatmap_h": 40,
+        "heatmap_w": 30,
+        "source": Source.CHIPPERV3,
+    },
```

Comment:
update the line 2 lines below so the default is "chipperv3"?

Reply:
Default mode is not Chipperv3